### ISSUE TITLE
Styling is not applied to the select list.

### ISSUE THREAD

**Issue by thirdknife** [JedWatson/react-select#618]
After installing through npm when I add this snippet 

```
<Select
     name="filter-category"
     value="one"
     options={options}
     onChange={logChange}
/>
```

I get a list but the styling is totally off. Can you help what's going wrong here.
![screen shot 2015-11-23 at 6 40 43 pm](https://cloud.githubusercontent.com/assets/885125/11337861/0a6464ba-9212-11e5-999f-00dfe8a143b2.jpg)

**Comment by bzums** [comment]
Are you importing the component like `import Select from 'react-select';` and using a bundler like Webpack? Then you have to import the CSS explicitly: `import 'react-select/dist/react-select.css';`

**Comment by bzums** [comment]
Otherwise this maybe interesting: https://github.com/JedWatson/react-select/issues/602

**Comment by thirdknife** [comment]
There was css loader missing with the https://github.com/christianalfoni/webpack-express-boilerplate webpack configuration 

```
    {
      test: /\.css$/,
      loader: 'style!css?modules&localIdentName=[name]---[local]---[hash:base64:5]',
      include: path.join(__dirname, 'app')
    },
    {
      test: /\.css$/,
      loader: 'style!css',
      include: path.join(__dirname, 'node_modules')
    }
```

Shouldn't it be loading by default. Is this an issue with the module it self or just the config?

**Comment by bzums** [comment]
Mhh maybe the following line has something to do with it:
https://github.com/JedWatson/react-select/blob/master/package.json#L6

Because this css file is not existing, currently?!

**Comment by thirdknife** [comment]
@bzums yes it's working for me with the loaders. I am closing this issue now.

**Comment by mattparrilla** [comment]
@thirdknife can you elaborate on why a loader is necessary? I'm having the same issue, loading the style manually works as a solution, but I'd prefer not to!

**Comment by thirdknife** [comment]
@mattparrilla it's because if you read https://github.com/JedWatson/react-select/blob/master/package.json line 6 it says 
`
"style": "dist/default.css",
`
and infact the dist folder does not contain default.css

**Comment by clinyong** [comment]
@bzums 

I import the component like `import Select from 'react-select';` and using a bundler like Webpack. And every time I have to add `import 'react-select/dist/react-select.css';`?

Is there another simple way?

**Comment by mrconnormurphy** [comment]
I''m using NPM and the CSS is missing - any ideas?

**Comment by JedWatson** [comment]
@mrconnormurphy you currently need to make sure you import the css - either the built `.css` file from `/dist` or you can include the `.less` or `.sass` versions into your own less or sass files (or import them directly with webpack loaders)

I'm thinking for simplicity of use, we might implement inline styles as an option, but that won't be for 1.0

**Comment by ciocan** [comment]
in your js component fil just import the scss
import 'react-select/scss/default.scss'

then in the webpack.config module loaders make sure to remove 
exclude: /node_modules/ 
beacause you will load the scss from that folder

**Comment by mrconnormurphy** [comment]
@JedWatson Thanks Jed - appreciate it - but I have different issues now.

I directly linked the CSS via my index.html file  
<link rel="stylesheet" type="text/css" href="node_modules/react-select/dist/react-select.css">

This seemed to fix the CSS issue. However I now have functionality issues as it seems like either some of the JS or CSS is still not loading / working correctly.

I can see the data in the drop down and it looks styled.
I can search by typing the country name etc... and this filtering works. 
However when I select an option nothing happens.

Any ideas?

**Comment by bntzio** [comment]
I had to copy the css from **node_modules/react-select/dist/react-select.css** into my own stylesheet to fix the css issue, and from there I can override some styles to fit my design.

**Comment by SVITY** [comment]
How to fix it https://github.com/JedWatson/react-select/issues/602#issuecomment-254835096

**Comment by adrianomic** [comment]
@JedWatson node_modules/react-select/dist/react-select.css this file does not exist in my node_modules directory.
How to fix it?
My react-select package version is 5.7.4

**Comment by honia19** [comment]
> @JedWatson node_modules/react-select/dist/react-select.css this file does not exist in my node_modules directory. How to fix it? My react-select package version is 5.7.4

https://github.com/JedWatson/react-select/issues/5710#issuecomment-1913643129
